middlewares/authn.sql

```diff
@@ -23,22 +23,8 @@ INSERT INTO enrollments
         course_instances AS ci
         JOIN pl_courses AS c ON (c.id = ci.course_id)
     WHERE
-        u.uid = $uid
-        AND c.short_name = $course_short_name
+        u.user_id = $user_id
+        AND c.example_course IS TRUE
 )
-ON CONFLICT DO NOTHING;
-
--- BLOCK insert_course_permissions_for_user
-INSERT INTO course_permissions
-    (user_id, course_id, course_role)
-(
-    SELECT
-        u.user_id, c.id, 'Previewer'::enum_course_role
-    FROM
-        users AS u,
-        pl_courses AS c
-    WHERE
-        u.uid = $uid
-        AND c.short_name = $course_short_name
-)
-ON CONFLICT DO UPDATE SET course_role = 'Previewer'::enum_course_role;
+ON CONFLICT DO NOTHING
+RETURNING *;
```

Comment:
Not that it matters, but we don't appear to actually use the return value so we could skip this final line.

Reply:
Good catch, I added this just to do some debugging. I'll remove it.

Reply:
Removed in 24ff048.